### PR TITLE
Updated README's, added whiteboard image and created Animal Shelter c…

### PR DESCRIPTION
**Pull request by littl3birdy**
- [x] Top-level README “Table of Contents” is updated
 - [ ] Feature tasks for this challenge are completed
 - [ ] Unit tests written and passing
     - [ ] “Happy Path” - Expected outcome
     - [ ] Expected failure
     - [ ] Edge Case (if applicable/obvious)
 - [x] README for this challenge is complete
     - [x] Summary, Description, Approach & Efficiency, Solution
     - [x] Link to code
     - [x] Picture of whiteboard

### CODE REVIEW

File: otherReadmes/animalShelter.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Feature:
+Create a class called AnimalShelter which holds only dogs and cats. The shelter operates using a first-in, first-out approach.
+Implement the following methods:
+* enqueue(animal): adds animal to the shelter. animal can be either a dog or a cat object.
+* dequeue(pref): returns either a dog or a cat. If pref is not "dog" or "cat" then return the name of the animal that has been in the queue the longest.
+
+
+# Approach and Efficiency:
+I estimate that this solution will have a time of O(n) and space of O(n).
```
Comment (SharinaS) [inline otherReadmes/animalShelter.md]: Time for Big O is more likely O(1), though it's a little hard to see in the image supplied.
